data/synthetic-blocks/synthetic-dataset-blocks.0.json

```diff
@@ -1 +1 @@
-{"metadata":{"type":"LBDatafile","rank":0},"phases":[{"id":0,"tasks":[{"entity":{"home":0,"id":1,"migratable":true,"type":"object"},"node":0,"resource":"cpu","time":0.5,"user_defined":{"shared_block_id":0,"shared_bytes":9.0,"home_rank":0}},{"entity":{"home":0,"id":3,"migratable":true,"type":"object"},"node":0,"resource":"cpu","time":0.5,"user_defined":{"shared_block_id":1,"shared_bytes":9.0,"home_rank":0}},{"entity":{"home":0,"id":2,"migratable":true,"type":"object"},"node":0,"resource":"cpu","time":0.5,"user_defined":{"shared_block_id":1,"shared_bytes":9.0,"home_rank":0}},{"entity":{"home":0,"id":0,"migratable":true,"type":"object"},"node":0,"resource":"cpu","time":1.0,"user_defined":{"shared_block_id":0,"shared_bytes":9.0,"home_rank":0}}],"communications":[{"type":"SendRecv","to":{"type":"object","id":5},"messages":1,"from":{"type":"object","id":0},"bytes":2.0},{"type":"SendRecv","to":{"type":"object","id":4},"messages":1,"from":{"type":"object","id":1},"bytes":1.0},{"type":"SendRecv","to":{"type":"object","id":2},"messages":1,"from":{"type":"object","id":3},"bytes":1.0},{"type":"SendRecv","to":{"type":"object","id":8},"messages":1,"from":{"type":"object","id":3},"bytes":0.5}]}]}
\ No newline at end of file
+{"metadata":{"type":"LBDatafile","rank":0},"phases":[{"id":0,"tasks":[{"entity":{"home":0,"seq_id":1,"migratable":true,"type":"object"},"node":0,"resource":"cpu","time":0.5,"user_defined":{"shared_block_id":0,"shared_bytes":9.0,"home_rank":0}},{"entity":{"home":0,"seq_id":3,"migratable":true,"type":"object"},"node":0,"resource":"cpu","time":0.5,"user_defined":{"shared_block_id":1,"shared_bytes":9.0,"home_rank":0}},{"entity":{"home":0,"seq_id":2,"migratable":true,"type":"object"},"node":0,"resource":"cpu","time":0.5,"user_defined":{"shared_block_id":1,"shared_bytes":9.0,"home_rank":0}},{"entity":{"home":0,"seq_id":0,"migratable":true,"type":"object"},"node":0,"resource":"cpu","time":1.0,"user_defined":{"shared_block_id":0,"shared_bytes":9.0,"home_rank":0}}],"communications":[{"type":"SendRecv","to":{"type":"object","seq_id":5},"messages":1,"from":{"type":"object","seq_id":0},"bytes":2.0},{"type":"SendRecv","to":{"type":"object","seq_id":4},"messages":1,"from":{"type":"object","seq_id":1},"bytes":1.0},{"type":"SendRecv","to":{"type":"object","seq_id":2},"messages":1,"from":{"type":"object","seq_id":3},"bytes":1.0},{"type":"SendRecv","to":{"type":"object","seq_id":8},"messages":1,"from":{"type":"object","seq_id":3},"bytes":0.5}]}]}
```

Comment:
I see that you updated the writer to add the entire `entity` field to the `communications`, but it doesn't look like the JSON files have been updated with this change.

This is only strictly required for datasets that we intend to also test in `vt`. For now, that's only `synthetic-blocks`, but I also think `ccm_example` might be relevant down the line.

Could you please regenerate the data files for those two problems?

Reply:
you are right the writer is OK but data is not. Its because I only renamed id to seq id in the data sets without regenerating it using the generator. I will update these data.

Reply:
It seems that only `synthetic-blocks` dataset contains its YAML specification.
We have to either edit manually or add a yaml file to `synthetic_lb_data` and `ccm_example`

Reply:
I think `synthetic-blocks` is the highest priority, since that's the test case we're using across the DARMA-tasking repos.

I'm not sure how the other two datasets were generated, but since there are no immediate plans to use them in vt, it might be okay to leave them as they are.

Reply:
It has been fixed for the 3 examples

Reply:
Fixed